app/controllers/discover.js

```diff
@@ -213,9 +213,19 @@ export default Ember.Controller.extend(Analytics, {
             return this.set('results', results);
         });
     },
+    totalPages: Ember.computed('numberOfResults', 'size', function() {
+        return Math.ceil(this.get('numberOfResults') / this.get('size'));
+    }),
+
     maxPages: Ember.computed('numberOfResults', function() {
         return ((this.get('numberOfResults') / this.get('size')) | 0) + (this.get('numberOfResults') % 10 === 0 ? 0 : 1);
     }),
+
+    clampedPages: Ember.computed('size', 'totalPages', function() {
+        let maxPages = Math.ceil(10000 / this.get('size'));
```

Comment:
Where does this number come from? What is a `clampedPage`? Why do we impose a maximum at all, instead of letting the API decide how many pages there really are?